includes/application/forwardApplication.php

```diff
@@ -63,26 +63,55 @@
 	
 	if( needApprover($status) )
 	{
-		echo <<<HTML
+		if( isset($_SESSION['PostName']) )
+		{
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->